src/Calendar.php

```diff
@@ -227,6 +228,33 @@ public function getTimestamps(): array
         return $this->timestamps;
     }
 
+    public function getEventStatusType(): EventStatusType
+    {
+        $eventStatusTypeCounts = [];
+        $eventStatusTypeCounts[EventStatusType::scheduled()->toNative()] = 0;
+        $eventStatusTypeCounts[EventStatusType::postponed()->toNative()] = 0;
+        $eventStatusTypeCounts[EventStatusType::cancelled()->toNative()] = 0;
```

Comment:
Clever 👏

Reply:
Although we could miss a new EventStatusType.

Reply:
No, adding a new type starts with adding a test, which will then fail anyway. It's good enough like this.

Reply:
We would probably miss a new `EventStatusType` with whatever approach to handle this, except by indeed adding a test. But this approach is very readable.